documentation-website/cypress/e2e/quick-start.cy.ts

```diff
@@ -11,12 +11,12 @@ describe('usage/quick-start', () => {
     cy.url().should('eq', BASE_URL + page.quickStart,);
   },);
   it('title matches', () => {
-    cy.title().should('eq', 'Quick Start | @idrinth/api-bench',);
+    cy.title().should('eq', 'Quick Start | @idrinth-api-bench/api-bench',);
   },);
   it('h1 matches', () => {
     cy.get('h1',)
       .invoke('text',)
-      .should('eq', '@idrinth/api-bench quick start',);
+      .should('eq', '@idrinth-api-bench/api-bench quick start',);
```

Comment:
this package will never exist, please revert the change for now

Reply:
Reverted